code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -281,3 +281,11 @@
 	name = "\improper Anti-Plague Sequence Omega autoinjector"
 	desc = "A small autoinjector containing 15 units of Anti-Plague Sequence Omega.  Cures all cases of the Necrotizing Plague. Also heals dead limbs."
 	list_reagents = list("zombiecure4" = 15)
+	
+/obj/item/reagent_containers/hypospray/autoinjector/hyper_medipen
+	name = "\improper suspicious medipen"
+	desc = "A cheap-looking medipen containing what seems to be a mix of nearly every medicine stored in the recently raided Nanotrasen warehouse." 
+	icon_state = "hyperpen"
+	amount_per_transfer_from_this = 45
+	volume = 45
+	list_reagents = list("salglu_solution" = 3, "synthflesh" = 4, "omnizine" = 3, "weak_omnizine" = 3, "perfluorodecalin" = 2, "sal_acid" = 1, "bicaridine" = 4, "kelotane" = 4, "epinephrine" = 5, "lavaland_extract" = 2, "rezadone" = 1, "spaceacillin" = 5, "teporone" =  2, "menthol" = 1, "iron" = 1, "sanguine_reagent" = 2, "vitamin" = 2)
```

Comment:
This list is very silly. It has way too many chemicals in it.

Unironically make a new medicine type just to put it in the pen that dumps a ton of healing into you but also has a super high metabolism rate.

Reply:
A new medicine type would allow someone to stack even more healing on top. It does look silly, but it also makes it much harder to abuse.

Reply:
> A new medicine type would allow someone to stack even more healing on top. It does look silly, but it also makes it much harder to abuse.

Counterpoint: It's only supposed to last for a few seconds, yes? The stacking won't matter.

Reply:
I guess it would work, but it feels like creating potential issues just for the sake of having a neater reagent list.

Reply:
You could also cheat and have the item directly heal you like a trauma kit.